### DISCUSSION TITLE
Finally a file manager that looks GOOD ✨

### DISCUSSION THREAD

**Discussion by milovangudelj** [spacedriveapp/spacedrive#1640] [General]
I'm a long time Linux user and seeing an app like Spacedrive being developed genuinely excites me. I'm also a web developer and UI designer and you did a damn good job so far! I love it!

Now for the downsides... It would be great if Spacedrive could automatically mount all drives attached to my machine. For example, I have a secondary SSD where I keep all my photos :camera_flash: and I can't see it from the app until I manually mount the drive using Nautilus and once I did that I still need root permissions to add it as a new location.

And lastly, I'm not really that used to AppImage files (my cup of tea is more like `.rpm`) and I never know where to put them or how to get it to show up as a desktop entry (something like `spacedrive.desktop`), but this is just a personal preference.

Again, great job and I'm so excited to see where this goes. :star_struck:

**Comment by HeavenVolkoff** [comment]
@milovangudelj Thank you for your kind words and support. Volume mounting is something we want to implement, and we are analyzing the feasibility and best approach on how to do it for every OS we support. Also, a `.rpm` release is on our roadmap: https://github.com/spacedriveapp/spacedrive/issues/1542#issuecomment-1760439040, we hope to get it ready for the beta release.
